src/__tests__/Streamable.js

```diff
@@ -0,0 +1,28 @@
+import cases from 'jest-in-case';
+
+import { getHTML, shouldTransform } from '../Streamable';
+
+cases(
+  'url validation',
+  ({ url, valid }) => {
+    expect(shouldTransform(url)).toBe(valid);
+  },
+  {
+    'non-streamable url': {
+      url: 'https://no-streamable-url-here.com',
+      valid: false,
+    },
+    'streamable-url-with-valid-embed-ending': {
+      url: 'https://streamable.com/bx960',
+      valid: true,
+    },
+  }
+);
+
+test('Builds the correct Streamable wrapper and iframe', async () => {
+  const html = await getHTML('https://streamable.com/bx960');
```

Comment:
If we keep doing the `fetch` call, we need to mock it in our tests

Reply:
Sure - if the API changes, though, the test would fail properly. Mocking it in the response will mask this. Your thoughts? I should mention that I'm a huge proponent of using mocking to prevent API calls; I didn't mock in this case because I saw there were other tests also making API calls.

Reply:
Twitter transformer tests are being mocked.
https://github.com/MichaelDeBoey/gatsby-remark-embedder/blob/8fd0afc46da27fb9d6d8dedb5541bf6e8785f686/src/__tests__/Twitter.js#L6-L15
https://github.com/MichaelDeBoey/gatsby-remark-embedder/blob/8fd0afc46da27fb9d6d8dedb5541bf6e8785f686/src/__tests__/plugin.js#L90-L97